package.json

```diff
@@ -4,9 +4,12 @@
   "private": true,
   "scripts": {
     "build": "tsc",
-    "start": "nodemon -e ts --exec \"(npm run lint -- --force && tsc && node build/server.js) || true\"",
+    "start": "npm run lint && npm run build && npm-run-all --parallel build:watch lint:watch serve:watch",
```

Comment:
`nodemon` restarts the server twice on startup. Minor, but annoying. Could this be avoided with [run-s](https://github.com/mysticatea/npm-run-all/blob/master/docs/run-s.md) ?